### PR TITLE
fix: cert issue

### PR DESCRIPTION
**Pull request by shdn-ibm**

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ DEFAULT_KUSTOMIZE_BIN="${OUTDIR_BIN}/kustomize"
 
 DEFAULT_IMAGE_REGISTRY="docker.io"
 DEFAULT_REGISTRY_NAMESPACE="ibmcom"
-DEFAULT_IMAGE_TAG="latest"
+DEFAULT_IMAGE_TAG="0.2.0"
```
Comment (lbscorpio) [inline hack/common.sh:64]: Don't update the default tag here, suggest to restore it, thanks.

**Review by lbscorpio — CHANGES_REQUESTED** [review]
If want to use another image tag, can use "export IMAGE_TAG=<new_tag>" method before building. Thanks.

File: hack/common.sh (hunk 1 above)
Comment (shdn-ibm) [reply hack/common.sh:64]: latest tag is not accept by certification

**Review by lbscorpio — APPROVED** [review]
/lgtm

**Pull request merged**